src/index.html

```diff
@@ -5,8 +5,386 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+     >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <img
+        class="header-logo"
+        src="./images/logo.png"
+        alt="MOYO"
+        >
```

Comment:
Need to check and fix it everywhere
```suggestion
      <img
        class="header-logo"
        src="./images/logo.png"
        alt="MOYO"
      >
```